### DISCUSSION TITLE
Error n forge test command.

### DISCUSSION THREAD

**Discussion by DevSwayam** [Cyfrin/foundry-full-course-cu#154] [Q&A] [answered]
On the timestamp https://youtu.be/sas02qSFZ74?t=1644 

Patrick have used forge test -m but when i am using the same command its showing  

``` Terminal
error: unexpected argument '-m' found

Usage: forge test [OPTIONS]

For more information, try '--help'.

```
and i used --help there is not an option for -m in their help commands so i am assuming that they have replaced it with other but there isnt any other command with same usecase.

**Comment by ShayanShamsi** [comment, marked as answer]
Hello @DevSwayam! -m has been replaced by --mt. Please write --mt instead of -m.

This has been mentioned in the chronological updates (check under Lesson 7) of the course [here](https://github.com/Cyfrin/foundry-full-course-f23/blob/main/chronological-updates.md).

**Reply by cromewar** [reply]
Thanks @ShayanShamsi

**Comment by rohit-bhalerao-7** [comment]
Use` forge test --mt <testfunction>`
